pom.xml

```diff
@@ -140,6 +140,11 @@
             <artifactId>lombok</artifactId>
         </dependency>
         <dependency>
+            <groupId>commons-codec</groupId>
```

Comment:
@longtimeago What is compile dependencies? I know only compile scope. How to deal with this dependency?